documentation/release-notes/topics/version_3.rst

```diff
@@ -1,6 +1,37 @@
 #########
 Version 3
 #########
+
+*****************************
+Hyperview 3.14 (June 7, 2023)
+*****************************
+This section covers significant changes and bug fixes in Hyperview 3.14.x since version 3.13.x.
+
+.. note:: This release has changes and improvements to the Data Collector software. It is strongly recommended to update your installed Data Collectors to the latest version to maintain an optimal monitoring and discovery experience.
+
+=======================================
+Enhanced Licensed Feature: Connectivity
+=======================================
+- RJ11 has been added as a connector type
+- Ports will be created automatically if they do not exist when importing connections
+- Users can now add Port 0 if needed
+- Port numbering will start at 0 for new network and network-connected devices, for example, network switches and servers
+
+==========================
+Other notable improvements
+==========================
+- Double-clicking on an asset in a Work Order will navigate the user to the asset
+- Sensor graph headings have been improved to make sensor names more readable. This impacts both the sensor dashboard and the sensor graph modal window
```

Comment:
This is a consistent improvement across the product where we display sensor names. For the sensor dashboard, this change affects the list view as well as the card view. This also impacts the sensor names in dashboard widgets and the power path Asset Info -> Sensors tab.